### PR TITLE
[Feat]: Sidechain review

### PR DESCRIPTION
**Pull request by phijfry**

### CODE REVIEW

File: tasks/deploy/gnosis-config.ts [hunk 1]
```diff
@@ -25,15 +25,15 @@ import { sidechainNaming } from "./sidechain-naming";
 
 const multisigs: SidechainMultisigConfig = {
     daoMultisig: "0x30019eB135532bDdF2Da17659101cc000C73c8e4", // Aura deployer EOA
-    pauseGuardian: ZERO_ADDRESS,
+    pauseGuardian: "0x30019eB135532bDdF2Da17659101cc000C73c8e4",
```
Comment (0xahtle7) [inline tasks/deploy/gnosis-config.ts:28]: Can you confirm the address?

File: tasks/deploy/gnosis-config.ts [hunk 2]
```diff
@@ -25,15 +25,15 @@ import { sidechainNaming } from "./sidechain-naming";
...
 };
 
 const extConfig: ExtSidechainConfig = {
-    canonicalChainId: 145, // https://layerzero.gitbook.io/docs/technical-reference/mainnet/supported-chain-ids#gnosis
+    canonicalChainId: 101, // https://layerzero.gitbook.io/docs/technical-reference/mainnet/supported-chain-ids
     lzEndpoint: "0x9740FF91F1985D8d2B71494aE1A2f723bb3Ed9E4", // https://layerzero.gitbook.io/docs/technical-reference/mainnet/supported-chain-ids#gnosis
     minter: ZERO_ADDRESS, // Mock minter
     token: "0x7eF541E2a22058048904fE5744f9c7E4C57AF717", // Mock token
-    create2Factory: ZERO_ADDRESS,
+    create2Factory: "0xf97De68aD9968A970aEf9849f2B9224506B7E1F6",
```
Comment (0xahtle7) [inline tasks/deploy/gnosis-config.ts:36]: Can we confirm the address?   this has to match in all chains

Comment (phijfry) [reply tasks/deploy/gnosis-config.ts:36]: There is a diff branch for gnosis config

**Review by 0xahtle7 — APPROVED** [review]
LGTM

**Pull request merged**